arch/arm/src/stm32/hardware/stm32_pinmap.h

```diff
@@ -34,7 +34,11 @@
 /* STM32L15xx family */
 
```

Comment:
missing pragma warning that a legacy pinmap is in use

Reply:
@raiden00pl I think I will move the #pragma to the gpio.c file so there is only one waring.